frontend/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { Tag as ChakraTag, TagProps as ChakraTagProps } from '@chakra-ui/react'
+
+type TagVariants = 'primary' | 'secondary'
```

Comment:
this should be exported from the theme file.

also, primary and secondary are current color schemes and thus may be confusing. Suggest "solid" and "light" variant

Reply:
Please do clarify with pearly as to whether we allow tags to be of other colors too. I think there is value in creating other colors (see current AngularJS beta tag)

Reply:
pearly mentioned 2 for now; might be more in the future. fwiw, she said it'll be in our colour palette so i'm tempted to leave it as is for now and when the additional colors come in, add them to the tag theming. wdyt?

Reply:
primary and secondary are not variants though, should still rename the variants into non-colorscheme related strings

Reply:
ps ya sorry for ambiguity; i agree with the theme export and i was just asking about colours!